test/WebJobs.Script.Tests.Integration/Host/ScriptHostManagerTests.cs

```diff
@@ -577,15 +605,11 @@ public void IsHostHealthy_ReturnsExpectedResult()
             hostManager.Stop();
             Assert.Equal(ScriptHostState.Default, hostManager.State);
 
-            await Task.Delay(FileTraceWriter.LogFlushIntervalMs);
-
-            string hostLogFilePath = Directory.EnumerateFiles(Path.Combine(logDir, "Host")).Single();
-            string hostLogs = File.ReadAllText(hostLogFilePath);
```

Comment:
Fixing flakiness I was seeing in file access. Simplifying test to get logs from the TestTraceWriter instead